sdk/log/logger.go

```diff
@@ -42,13 +43,30 @@ func (l *logger) Emit(ctx context.Context, r log.Record) {
 	}
 }
 
-func (l *logger) Enabled(ctx context.Context, r log.Record) bool {
-	newRecord := l.newRecord(ctx, r)
-	for _, p := range l.provider.processors {
-		if enabled := p.Enabled(ctx, newRecord); enabled {
+// Enabled returns true if at least one Processor held by the LoggerProvider
```

Comment:
> at least one Processor

Does this mean that the minsev processor won't actually work?  For example, if someone uses the batch processor and the minsev processor, the batch processor will process the record, but the minsev processor won't, this will return true, right?

Reply:
> Does this mean that the minsev processor won't actually work?

I'm not sure it's fair to say the minsev processor won't work. If both processors are registered with the `LoggerProvider` directly, given the batch processor will always return true, the minsev processor will not have an effect. That is correct, and also expected behavior.  However, the minsev processor is a wrapping processor that you would not normally register directly with `LoggerProvider`.

If you want to gate the registered batch processor with the minsev processor you would wrap the batch processor in the minsev processor. Registering them independently in the `LoggerProvider` means they should be considered separate pipelines and the `LoggerProvider` should consult each to determine if any are enabled and report so if discovered.

Reply:
The mindset processor would still filter out records by the `OnEmit` method even if other processors' `Enabled` methods return True.

https://github.com/open-telemetry/opentelemetry-go-contrib/blob/af75717ac4fb3ba13eaea83b88301723122060cf/processors/minsev/minsev.go#L47-L52